resteasy-spring-feature-pack/common/src/main/resources/layers/standalone/resteasy-spring/layer-spec.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:2.0" name="resteasy-spring">
```

Comment:
@jfdenise With my understanding this `layer` concept is different from the `layer` in Galleon right?

Reply:
That is a Galleon layer, exactly the same concept.

Reply:
Okay!